main/Main.hs

```diff
@@ -32,11 +32,7 @@ import System.IO                (BufferMode (..), hSetBuffering)
 import Text.Printf              (hPrintf)
 
 import PostgREST.App         (postgrest)
-import PostgREST.Config      (AppConfig (..), CLI (..), Command (..),
-                              Environment, configDbPoolTimeout',
-                              dumpAppConfig, prettyVersion,
-                              readCLIShowHelp, readEnvironment,
-                              readValidateConfig)
+import PostgREST.Config
```

Comment:
We should avoid unqualified wildcard imports where possible, as they make the code more difficult to follow. 

But I think we can leave it like this for now, I was going to make a suggestion anyway for refactoring our imports and untangling some of the code (esp. App.hs).

Reply:
I agree. I wanted to split up the import, to import `Environment` from `PostgREST.Config` only in the `#ifndef ...` block below - but that throws off the linter, because hlint wants to merge those two import statements then...

Qualifying it would make it easier to understand - but that's something we should tackle on a bigger scale, as you say.

Reply:
I think the problem here is just the `Environment` import.

Couldn't we wrap that import with `#ifndef mingw32_HOST_OS` and leave the rest explicit?

Reply:
> Couldn't we wrap that import with `#ifndef mingw32_HOST_OS` and leave the rest explicit?

Cross-posted here, but see above: hlint complains about that. The unqualified import was the only way out I found here that fixes the build error for now, but does not involve more refactoring across the whole file.

Reply:
Actually I remembered it wasn't that simple because of `hlint`.

That's why I had to make the `UnixSocket.hs` file. So I agree with leaving it like this for now.

Reply:
> Cross-posted here, but see above: hlint complains about that.

Ah, you beat me to it(non real-time comments played me). Yes, the unqualified import is good for now.